File: 1635-number-of-good-pairs/solution.java
class Solution {
    public int numIdenticalPairs(int[] nums) {
        int c=0;
        int n=nums.length;
        for(int i=0;i<n-1;i++)
        {
            for(int j=i+1;j<n;j++)
            {
                if(nums[i]==nums[j])
                    c++;
            }
        }
        return c;
    }
}

File: 1635-number-of-good-pairs/solution.cpp
class Solution {
public:
    int numIdenticalPairs(vector<int>& nums) {
        unordered_map<int,int> map;
        int c=0;
        for(auto ele:nums)
        {
            c+=map[ele];
            map[ele]+=1;
        }
        return c;
    }
};
